### PR TITLE
selfies release and fixed vocab shipping

### LINKED ISSUE #81: selfies 1.0 release
our tests for selfies tokenization fail for newer versions of the `selfies` dependency.
```
FAIL: test_tokenize_selfies (pytoda.smiles.tests.test_processing.TestProcessing)
Test tokenize_selfies.
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/Users/dow/public_dev/paccmann_datasets/pytoda/smiles/tests/test_processing.py", line 142, in test_tokenize_selfies
    self.assertListEqual(tokenize_selfies(selfies), ground_truth)
AssertionError: Lists differ: ['[C]', '[C]', '[=N]', '[O]', '[C]', '[Expl=Ring1]', '[Branch1_1]'] != ['[c]', '[c]', '[n]', '[o]', '[c]', '[Ring1]', '[Ring2]']

First differing element 0:
'[C]'
'[c]'

- ['[C]', '[C]', '[=N]', '[O]', '[C]', '[Expl=Ring1]', '[Branch1_1]']
?    ^      ^      ^^      ^      ^      -----           ^^^ ^^^^^

+ ['[c]', '[c]', '[n]', '[o]', '[c]', '[Ring1]', '[Ring2]']
?    ^      ^      ^      ^      ^                 ^^ ^^


```
we should require a version  >1 in setup.py, pin an up to date version in requirements.txt and adapt the tests for correct selfies tokens. 
See https://github.com/aspuru-guzik-group/selfies/blob/master/CHANGELOG.md

Besides these obvious task, there is now [`selfies.utils.split_selfies`](https://github.com/aspuru-guzik-group/selfies/blob/9f822e175e559fa11753b715991645afe9953621/selfies/utils.py#L25).
Could it replace our implementation of `tokenize_selfies` altogether @jannisborn ?

### PR DESCRIPTION
**Pull request by C-nit**
- Updated tests for release version of selfies package.
 - Added deprecation warning to `tokenize_selfies` in favor of offical `split_selfies`, that is the new selfies default.
The above resolve #81.
- Fixed package data inclusion for pretrained tokenizer.
- Minor refactor in `ProteinFeatureLanguage`.

### CODE REVIEW

**Comment by C-nit** [comment]
I updated the version already, but we need to tag after merge still.

**Comment by jannisborn** [comment]
This will close #80

File: pytoda/proteins/protein_feature_language.py [hunk 1]
```diff
@@ -83,25 +96,6 @@ def __init__(
...
-                    [
-                        self.index_to_token.get(token_index, '')
-                        for token_index in token_indexes
-                        if token_index in self.sequence_tokens
-                    ]
-                )
-            )
-        else:
-            # Lambda function that throws an exception.
-            self._token_indexes_to_sequence = lambda x: (_ for _ in ()).throw(
```
Comment (jannisborn) [inline pytoda/proteins/protein_feature_language.py:98]: You destroyed my beautiful lambda function that can throw exceptions 😢😄 👍

File: pytoda/smiles/tests/test_processing.py [hunk 2]
```diff
@@ -116,30 +117,55 @@ def test_spe_smiles_tokenizer(self) -> None:
...
+                    '[Ring1]', '[Branch1_2]'
                 ]
             )
         ]:
+            self.assertListEqual(
+                # list wrapping version
+                split_selfies(sf.encoder(smiles)), ground_truth
+            )
+
+    def test_tokenize_selfies_match(self) -> None:
```
Comment (jannisborn) [inline pytoda/smiles/tests/test_processing.py:161]: that's a great one

**Review by jannisborn — APPROVED** [review]
Happy to see that the native `split_selfies` and our `tokenize_selfies` are doing the same.

**Pull request merged**